pkg/controller/storagecluster/pxrepo.go

```diff
@@ -0,0 +1,121 @@
+package storagecluster
```

Comment:
Can we move this under components package in 'driver/storage/portworx'?

Reply:
fixed.